File: content/api-documentation/how-to/account/examples/accountInfo.js
const financefeast = require('@financefeasthq/financefeast-trade-api')
const financefeast = new financefeast()

// Get our account information.
financefeast.getAccount()
    .then((account) => {
        // Check if our account is restricted from trading.
        if (account.trading_blocked) {
            console.log('Account is currently restricted from trading.')
        }

        // Check how much money we can use to open new positions.
        console.log(`$${account.buying_power} is available as buying power.`)
    })

File: content/api-documentation/how-to/account/examples/accountInfo.go
package main

import (
	"fmt"

	"github.com/financefeasthq/financefeast-trade-api-go/financefeast"
)

func init() {
	financefeast.SetBaseUrl("https://paper-api.financefeast.markets")
}

func main() {
	// Get our account information.
	account, err := financefeast.GetAccount()
	if err != nil {
		panic(err)
	}

	// Check if our account is restricted from trading.
	if account.TradingBlocked {
		fmt.Println("Account is currently restricted from trading.")
	}

	// Check how much money we can use to open new positions.
	fmt.Printf("%v is available as buying power.\n", account.BuyingPower)
}